src/Sylius/Bundle/CoreBundle/Resources/config/test_services.xml

```diff
@@ -41,6 +42,10 @@
             <argument type="service" id="payum" />
         </service>
 
+        <service id="sylius.behat.email_checker" class="%sylius.behat.email_checker.class%">
```

Comment:
It's only usable in tests though, also what about [this](https://github.com/Sylius/Sylius/pull/5029/files#diff-a28d152be9aaa542ba55bf77d3ec7236L25) etc?